blog_system/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from django.urls import path, include
+from debug_toolbar.toolbar import debug_toolbar_urls
```

Comment:
It seems like there is an error in the import statement. The correct way to import debug_toolbar_urls is from 'debug_toolbar' module, not from 'debug_toolbar.toolbar'. The correct import statement should be 'from debug_toolbar import debug_toolbar_urls'.